medusa/search/core.py

```diff
@@ -729,7 +729,9 @@ def search_providers(series_obj, episodes, forced_search=False, down_cur_quality
                           best_season_result.provider.provider_type,
                           best_season_result.name)
             else:
-                if best_season_result.provider.provider_type == GenericProvider.NZB:
+                # Some NZB providers (e.g. Jackett) can also download torrents, but torrents cannot be split like NZB
+                if (best_season_result.provider.provider_type == GenericProvider.NZB and
```

Comment:
This shouldn't be needed since the Jackett changes, as there are no more NZB providers that can download torrents.

Reply:
You are correct, and I thought about it.
However people still use Newznab for Jackett and they just get that error with no explanation.
You did obviously add a deprecated warning, but I guess it would take some time.

Reply:
True that, but we should remove all these workarounds at some point.